### ISSUE TITLE
🚧Add multiple PR reviewers during PR issue sync

### ISSUE THREAD

**Issue by CalvinWilkinson** [KinsonDigital/Infrastructure#184]
### Complete The Item Below

- [X] I have updated the title without removing the 🚧 emoji.

### Description

During the PR to issue sync process, add the ability to add multiple reviewers to the PR instead of only one.

Currently, the value of the `DEFAULT_PR_REVIEWER` repository variable is used as the GitHub user name to add as a reviewer.  This is only a single name and will not work for multiple names.  

To add multiple names, update the _**SyncPRToIssueRunner.ts**_ class code to process the `DEFAULT_PR_REVIEWER` value as a comma-delimited list.  If there is only a single item, process as normal.  If multiple reviewers should be added, create an array of names to send into the `PullRequestClient.requestReviewers()` function to request the list of users.

Go to the [pull api docs](https://docs.github.com/en/rest/pulls/review-requests?apiVersion=2022-11-28#request-reviewers-for-a-pull-request) for more info.

> **Note**
> Currently, the function `PullRequestClient.requestReviewers()` does not exist. The function `PullRequestClient.requestReviewer()` does.  This function needs to be updated to have a plural name and the `reviewer` parameter updated to be a single string or array of strings.  This work will be done with issue 4444 and needs to be done first.

### Acceptance Criteria

- [ ] The _**SyncPRToIssueRunner.ts**_ class updated for new type of repo variable value

### ToDo Items

- [X] Change type labels added to this issue.  Refer to the _**Change Type Labels**_ section below.
- [X] Priority label added to this issue.  Refer to the _**Priority Type Labels**_ section below.
- [ ] Issue linked to the correct milestone _(if applicable)_.

### Issue Dependencies

- #175
- KinsonDigital/kd_clients#11
    > **Note**
    > This one is important. This will contain the client change to send in an array of reviewers to request for a PR

### Additional Information:

**_<details closed><summary>Change Type Labels</summary>_**

| Change Type         | Label                |
|---------------------|----------------------|
| Bug Fixes           | `🐛bug` |
| Breaking Changes    | `🧨breaking changes` |
| New Feature         | `✨new feature` |
| CICD Changes        | `♻️cicd` |
| Config Changes      | `⚙️config` |
| Code Doc Changes    | `🗒️documentation/code` |
| Product Doc Changes | `📝documentation/product` |
</details>

**_<details closed><summary>Priority Type Labels</summary>_**

| Priority Type       | Label             |
|---------------------|-------------------|
| Low Priority        | `low priority`    |
| Medium Priority     | `medium priority` |
| High Priority       | `high priority`   |
</details>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.
